src/responsibleai/rai_analyse/create_rai_insights.py

```diff
@@ -177,7 +188,7 @@ def main(args):
         args.output_path, DashboardInfo.RAI_INSIGHTS_PARENT_FILENAME
     )
     with open(output_file, "w") as of:
-        json.dump(output_dict, of)
+        json.dump(output_dict, of, default=feature_metadata_to_dict)
```

Comment:
Do we want to consider supporting json encoding in the feature metadata class itself in future release?
Also, could default handler be updated to something more generally named for example default_json_handler and then add custom logic for handling featuremetadata type?

Reply:
1. I think it's better to handle it here instead of inside feature metadata class. Because in the case, we pass a class type instead of a dict, which is more clear.
2. yes will update to a more generic name!